### DISCUSSION TITLE
protostar/docs/tutorials/configuration-file

### DISCUSSION THREAD

**Discussion by giscus** [software-mansion/protostar#1148] [Documentation]
# protostar/docs/tutorials/configuration-file

Protostar can be configured using a configuration file called protostar.toml, which is placed in the root of your project.

https://docs.swmansion.com/protostar/docs/tutorials/configuration-file

**Comment by hypekn1ght** [comment]
on 0.7.0 users migrating from v1 to v2, needs to manually edit 'cairo_path' to 'cairo-path'
